cli/lockfile.rs

```diff
@@ -42,13 +45,25 @@ impl Lockfile {
     }
     let j = json!(&self.map);
     let s = serde_json::to_string_pretty(&j).unwrap();
+    let s_options = FmtOptionsConfig {
+      use_tabs: Some(false),
+      line_width: Some(80),
+      indent_width: Some(2),
+      single_quote: Some(true),
+      prose_wrap: Some(ProseWrap::Preserve),
+    };
+    let formated_s = format_json(&s, &s_options);
+    let formated_str = match formated_s {
+      Ok(x) => x,
+      Err(e) => e,
```

Comment:
We can't do this because this would write out an error message to the file. Maybe consider something like:

```ts
let s = format_json(&s, &Default::default).unwrap_or_else(|| s.to_string());
```

That way it falls back to the original string when the formatting fails for some reason.